packages/x-date-pickers/src/DesktopTimePicker/DesktopTimePicker.tsx

```diff
@@ -77,9 +77,8 @@ const DesktopTimePicker = React.forwardRef(function DesktopTimePicker<
   };
 
   const ampmInClock = defaultizedProps.ampmInClock ?? true;
-  const actionBarActions: PickersActionBarAction[] = shouldRenderTimeInASingleColumn
-    ? []
-    : ['accept'];
+  const actionBarActions: PickersActionBarAction[] = ['cancel', 'accept'];
```

Comment:
I don't have the strictest opinion on this one, but given that we will have only 2 components without actions, I thought it would make the most sense to simplify this management and put it where it makes the most sense. 🤔 
If we look at the problem like this: should the layout logic care about or manage the necessary actions? 🤔 
WDYT?